swap/protocol_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/ethereum/go-ethereum/p2p"
 	"github.com/ethereum/go-ethereum/p2p/enode"
 	"github.com/ethereum/go-ethereum/rpc"
-	contract "github.com/ethersphere/swarm/contracts/swap"
```

Comment:
I would prefer that we rename the package `contracts/swap` to `contracts/chequebook`, then we don't have to do this alias here.